metpy/calc/thermo.py

```diff
@@ -847,6 +847,38 @@ def mixing_ratio_from_specific_humidity(specific_humidity):
 
 
 @exporter.export
+@check_units('[dimensionless]')
+def specific_humidity_from_mixing_ratio(mixing_ratio):
+    r"""Calculate the specific humidity from the mixing ratio.
+
+    Parameters
+    ----------
+    mixing_ratio: `pint.Quantity`
+        mixing ratio
+
+    Returns
+    -------
+    `pint.Quantity`
+        Specific humidity
+
+    Notes
+    -----
+    Formula from [Salby1996]_ pg. 118.
+
+    .. math:: q = \frac{w}{1+w}
+
+    * :math:`w` is mixing ratio
+    * :math:`q` is the specific humidity
+
+    See Also
+    --------
+    mixing_ratio
+
```

Comment:
Should probably include: `mixing_ratio_from_specific_humidity`.

Would also be good to update the docstring for that function to point here as well.